s3transfer/processpool.py

```diff
@@ -20,10 +20,22 @@
 from s3transfer.constants import MB
 from s3transfer.exceptions import RetriesExceededError
 from s3transfer.utils import S3_RETRYABLE_DOWNLOAD_ERRORS
+from s3transfer.utils import calculate_num_parts
+from s3transfer.utils import calculate_range_parameter
 
 logger = logging.getLogger(__name__)
 
 SHUTDOWN_SIGNAL = 'SHUTDOWN'
+DownloadFileJob = collections.namedtuple(
+    'DownloadFileJob', [
+        'transfer_id',    # The unique id for the transfer
+        'bucket',         # The bucket to download the object from
+        'key',            # The key to download the object from
+        'filename',       # The user-requested download location
+        'extra_args',     # Extra arguments to provide to client calls
+        'expected_size',  # The user-provided expected size of the download
+    ]
+)
 GetObjectJob = collections.namedtuple(
```

Comment:
I'm hesitant to call it `GetObjectPartJob` because the `GetObjectWorker` is not necessarily doing a ranged download. Would naming it something like `DownloadFileSubmitterJob` and the other one `GetObjectWorkerJob` help? Once the `ProcessPoolDownloader` class is fleshed out, I'd imagine it will be a bit easier to follow.